stlab/copy_on_write.hpp

```diff
@@ -26,10 +26,12 @@ class copy_on_write {
     struct model {
         std::atomic<std::size_t> _count{1};
 
-        model() noexcept(std::is_nothrow_constructible_v<T>) = default;
+        model() noexcept(std::is_nothrow_constructible<T>::value) = default;
```

Comment:
I presume you're following some guideline/convention that's documented somewhere?

Reply:
Required for C++14 - `_v` is a C++17 addition.